src/test/java/andersen/AndersenTasks/collections/CustomArrayListTests.java

```diff
@@ -0,0 +1,55 @@
+package andersen.AndersenTasks.collections;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+
+public class CustomArrayListTests {
+    private CustomArrayList customArrayList;
+
+    @BeforeEach
+    public void init(){
+        customArrayList = new CustomArrayList<>();
+    }
+
+    @Test
+    public void testNotNull(){
```

Comment:
You should test your business logic not java itself. Cause after you initialize customArrayList list with new operator it always will be not null.